tests/browser/contacts_accounts/add_existingcontact_household.robot

```diff
@@ -16,5 +16,7 @@ Add Existing Contact to Existing Household
     Populate Lookup Field    Account Name    &{contact1}[LastName] Household
     Click Modal Button    Save
     Click Header Field Link  Account Name
+    ${id} Get Current Record Id
```

Comment:
Should be `${id} =`

Reply:
this seems to be working, any specific reason why it should be specified that way?

Reply:
forgot to give extra space and that's why it failed.

Reply:
Ohhh that makes sense.